helm-chart/binderhub/values.yaml

```diff
@@ -17,7 +17,7 @@ nodeSelector: {}
 
 image:
   name: jupyterhub/k8s-binderhub
-  tag: 'local'
+  tag: local
```

Comment:
If you use a round trip, the quotes may not be left for a string that needs them i think.

Retainig explicit quotes for the tag has been relevant in the past i recall, perhaps not by chartpress generates image tags, but i'm still cautious preferring to see us having the yaml include quotes foe our placeholder values where they could be needed.

Reply:
The quotes were removed _by_ the roundtrip serialization. I did not remove them. I left the change here because every re-write of the file will strip it.

Reply:
Ah, this is how we have configured ruamel.yaml in chartpress to avoid this kind of issue.

```python
# use safe roundtrip yaml loader capable or preserving usage of no/single/double
# quotes for a string for example
yaml = YAML(typ="rt")
yaml.preserve_quotes = True
yaml.indent(mapping=2, offset=2, sequence=4)
```

The indent settings is relevant to be compatible with prettier formatting, at least if not having a rout-tripped format to retain.

```yaml
exampleList:
- without-indent-settings

# this is the kind of formatting we systematically
# use and enforce with prettier, and requires explicit
# configuration of ruamel.yaml at least when generating
# new yaml files, but I'm uncertain if it's required when
# round-tripping on existing files with existing list entries
anotherExampleList:
  - with-indent-settings
```

Reply:
Got it! I remembered the indent setting (it's the same here), but forgot the quotes. Will fix.

Reply:
Fixed.